docs/_main/getting_started.md

```diff
@@ -82,12 +78,16 @@ If you see the same output as above, we're ready to start.
 In this guide we'll post a simple message that contains the current time. We'll also follow the best practice of keeping
 secrets outside of your code (do not hardcode sensitive data).
 
+Before we move forward, add the bot user you created above to the `#general` channel in your workspace. Bots need to be
+invited to channels to be able to post in them. You can do this by going to the `#general` channel inside slack in your workspace and
```

Comment:
A suggestion you can take or leave: maybe add a note about `chat:write:public` here in parenthesis